src/sections/support-page/global-enterprise/global-enterprise.tsx

```diff
@@ -0,0 +1,134 @@
+import React, { HTMLAttributes } from 'react';
+import classNames from 'classnames';
+
+import { Paragraph } from '@bit/bit.base-ui.text.paragraph';
+import { PossibleSizes } from '@bit/bit.base-ui.theme.sizes';
+import { text, marginCenter } from '@bit/bit.base-ui.layout.align';
+import { Card, CardProps } from '@bit/bit.base-ui.surfaces.card';
+import { Grid } from '@bit/bit.base-ui.layout.grid-component';
+import { centerColumn } from '@bit/bit.base-ui.layout.page-frame';
+
+import { Button } from '../../../elements/button';
+import { H3, H2 } from '../../../elements/heading';
+import { Image } from '../../../elements/image';
+
+import styles from './global-enterprise.module.scss';
+import { Link } from '../../../elements/link';
+
+type GlobalEnterpriseProps = {} & HTMLAttributes<HTMLElement>;
+
+/**
+ * Opening section for the Support offering page.
+ * @name EcoSystem
+ */
+export function GlobalEnterprise(props: GlobalEnterpriseProps) {
+	const cards = [
+		{
+			iconUrl: 'support-page/support.svg',
+			title: 'Response time SLA',
+			text:
+				"We'll respond quick and with effective solutions to make sure you never slow down.",
+			alt: '',
```

Comment:
please give alt text

Reply:
ill ask yoni if there is anything specific we want as the alt text or we can just use the image name